### ISSUE TITLE
`@everyone`

### ISSUE THREAD

**Issue by SoniEx2** [mastodon/mastodon#19923]
### Pitch

`@everyone` sends a notification to all your followers

### Motivation

useful when moving to other networks etc

**Comment by Dremor** [comment]
Shouldn't it be `@followers` instead ? `@everyone` imply a very broad broadcast, aka everyone in an instance, and could confuse users.

**Comment by SoniEx2** [comment]
sure we guess

**Comment by jonaharagon** [comment]
I could see this being useful for instance admins to `@everyone` on their instance, I don't see how allowing anyone to `@followers` all their followers would provide any value, isn't that what the Home timeline is for?

**Comment by SoniEx2** [comment]
not really. users should have a notification channel for all of their followers. followers can turn it off individually. would be useful for all sorts of things.

**Comment by rbairwell** [comment]
Do you mean the already supported [Followers-Only](https://docs.joinmastodon.org/user/posting/#private) publishing level?

**Comment by SoniEx2** [comment]
no, `@everyone` would show up in the notifications/mentions tab.

the post itself can be public or unlisted.
